src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,35 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.10.0
+5 July 2023
+
```

Comment:
I think we have to include a `removal notice` for `1.23` like we did [here](https://www.okteto.com/docs/self-hosted/install/releases/#removal-notice) for `1.21`.

We didn't do it for `1.22` but I think it is good to explicitly call it out. WDYT @omnomagonz ?